sdl_android/src/main/java/com/smartdevicelink/proxy/rpc/GPSData.java

```diff
@@ -181,30 +181,10 @@ public GPSData(Hashtable<String, Object> hash) {
     /**
      * Constructs a newly allocated GPSData object
      */

```

Comment:
Removing this old constructor is a major change. Please keep it and deprecate it.